internal/templates/hello-world.neva

```diff
@@ -0,0 +1,7 @@
+import { fmt }
```

Comment:
Please move to `internal/cli/templates`, this and the next one

Reply:
Maybe I should just move all the templates to internal\cli\templates?

Reply:
Yes, that's was my intention :)